### PR TITLE
Update review collapsible titles and add tooltip to info icons

### PR DESCRIPTION
**Pull request by calvinlu3**
This fixes https://github.com/oncokb/oncokb-pipeline/issues/374

### Changes
1. Add tooltip to info icons
info-tooltip-pr

2. Collapsible title updates
image

3. Gene Type review
a. Uncheck ocg and check tsg 
image
b. Uncheck ocg 
image
c. Check both
image
d. Uncheck both
image

### CODE REVIEW

File: src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx [hunk 1]
```diff
@@ -202,6 +228,55 @@ export const ReviewCollapsible = (props: IReviewCollapsibleProps) => {
...
+  const getMultiSelectionReviewContent = () => {
+    const multiSelectionReviewLevel = props.baseReviewLevel as MultiSelectionReviewLevel;
+    const joinedNewParts = [];
+    const joinedOldParts = [];
+    for (const reviewLevel of multiSelectionReviewLevel.getReviewLevels()) {
+      joinedNewParts.push(reviewLevel.currentVal);
+      joinedOldParts.push(reviewLevel.reviewInfo.lastReviewedString);
+    }
+    /* eslint-disable no-console */
+    console.log(joinedNewParts, joinedOldParts);
```
Comment (zhx828) [inline src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx:240]: do we need this?

File: src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx [hunk 2]
```diff
@@ -202,6 +228,55 @@ export const ReviewCollapsible = (props: IReviewCollapsibleProps) => {
...
+    return (
+      <div className="mb-2">
+        <ReactDiffViewer
+          styles={REACT_DIFF_VIEWER_STYLES}
+          showDiffOnly
+          extraLinesSurroundingDiff={0}
+          oldValue={joinedOldParts.filter(p => p !== '').join('\t')}
+          newValue={joinedNewParts.filter(p => p !== '').join('\t')}
```
Comment (zhx828) [inline src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx:248]: I wonder whether tab is the best way to join or use comma.

File: src/main/webapp/app/pages/curation/nestLevelSummary/NestLevelSummary.tsx [hunk 3]
```diff
@@ -20,77 +20,112 @@ export type NestLevelSummaryStats = {
...
+const levelSummaryKeys: (keyof NestLevelSummaryStats)[] = ['dxLevels', 'pxLevels', 'txLevels'];
+
+const READABLE_SUMMARY_KEY: { [key in keyof NestLevelSummaryStats]: string } = {
+  TT: 'tumor type(s)',
+  TTS: 'Tx summary',
+  DxS: 'Dx summary',
+  PxS: 'Px summary',
+  dxLevels: 'Dx level',
+  pxLevels: 'Px level',
+  txLevels: 'Level',
```
Comment (zhx828) [inline src/main/webapp/app/pages/curation/nestLevelSummary/NestLevelSummary.tsx:33]: May as well to use `Tx Level`

**Review by zhx828 — APPROVED** [review]
Some minor comments

File: src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx (hunk 2 above)
Comment (calvinlu3) [reply src/main/webapp/app/pages/curation/collapsible/ReviewCollapsible.tsx:248]: Comma looks better especially if we have more than two selections like for allele state
image

**Pull request merged**